doc/changelog/06-Ltac2-language/18102-ltac2-setoid-rewrite.rst

```diff
@@ -0,0 +1,4 @@
+- **Added:**
+  ``setoid_rewrite`` tactic
```

Comment:
```suggestion
  Support for the ``setoid_rewrite`` tactic
```

The tactic is not new, just the Ltac2 interface to it.

Reply:
indeed, thanks